README.md

```diff
@@ -41,7 +41,7 @@ sure the right thing happened. It's also a very good idea to run
 `alembic downgrade -1` to back up to before your revision and check that the
 database looks right (of course you then need to re-run the upgrade command to
   get back to where you meant to be.)
-5. Run `nosetests` to check that all the tests pass.
+5. Run `pytest` to check that all the tests pass.
```

Comment:
to just run `pytest` as a bare command from inside the folder, I would recommend adding a setup.cfg file and defining folders to be skipped like scripts. Pyuvdata has a decent example of a started setup.cfg with a pytest section too.

Reply:
I just run `pytest` in the folder and it works fine...

Reply:
okay, may just have been a poorly named script in an older repo. probably called "test_something" and pytest tried to execute it.

Reply:
I think the setup.cfg file is only needed if you want to use setup to run the tests, right?

Reply:
it can be used for both. an aliases section is to be able to run `python setup.py test` and there is a tools:pytest section that can be used to pass extra or desired default arguments to pytest

Reply:
ok. I can make that file easily.